src/styles/sidebar/sidebar.scss

```diff
@@ -115,51 +115,10 @@ hypothesis-app {
   }
 }
 
-// Elements in root template (viewer.html)
-// ---------------------------------------
-
-.sheet {
```

Comment:
The `FocusedModeHeader` component still references the `sheet` and `sheet--short` classes.

Reply:
Facepalm. Bad grep methodology.